src/browser/preboot_browser.ts

```diff
@@ -22,13 +22,13 @@ import {
 } from '../preboot_interfaces';
 
 // the idea here is that preboot is a global value on the window that is used by the client
-var preboot = prebootClient();
```

Comment:
you are right. preboot variable is required.
I think this ts file has a es2015 module syntax, this variable is not required.
But it is wrong. import statement is removed at compile time.

`<script src="preboot_client.min.js"></script>`
In this case, this variable is visible from global namespace.